src/OpenTelemetry.Instrumentation.SqlClient/Implementation/SqlClientInstrumentationEventSource.cs

```diff
@@ -75,5 +75,17 @@ public void EnrichmentException(string exception)
         {
             this.WriteEvent(5, exception);
         }
+
+        [Event(6, Message = "Command is filtered out.", Level = EventLevel.Verbose)]
+        public void CommandIsFilteredOut(string eventName)
+        {
+            this.WriteEvent(6, eventName);
+        }
+
+        [Event(7, Message = "Command filter threw exception. Request will not be collected. Exception {0}.", Level = EventLevel.Error)]
+        public void CommandFilterException(string exception)
```

Comment:
use [this](https://github.com/open-telemetry/opentelemetry-dotnet/blob/a6c3cac15d20cb39545c0a564ca0595fada607ee/src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/AspNetCoreInstrumentationEventSource.cs#L31-L38) pattern.

Reply:
@vishweshbankwar updated as per sample, I hope